__docs__/wonder-blocks-dropdown/multi-select.accessibility.stories.tsx

```diff
@@ -129,3 +131,42 @@ export const UsingCustomOpenerAriaLabel = {
     render: MultiSelectCustomOpenerLabel.bind({}),
     name: "Using aria-label on custom opener",
 };
+
+const optionItems = allCountries.map(([code, translatedName]) => (
+    <OptionItem key={code} value={code} label={translatedName} />
+));
+
+const MultiSelectWithVisibleAnnouncer = () => {
+    // React.useEffect(() => {
+    //     // Inject Announcer into the Storybook iframe instead of body
+    //     const storybookRoot = document.getElementById("storybook-docs");
+    //     if (storybookRoot) {
+    //         console.log("announcer init");
+    //         // initAnnouncer({targetNode: storybookRoot});
+    //     }
+    // }, []);
```

Comment:
How did this go? Did using the Storybook decorator at the root help with the issues you were seeing?